app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,16 @@
     package="fr.tabbya.instantvisio">
 
     <uses-permission android:name="android.permission.SEND_SMS"/>
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS" />
+    <uses-permission android:name="android.permission.RECORD_AUDIO" />
+    <uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS" />
```

Comment:
duplicate line

Reply:
isn't `camera` & `microphone` enough ? If the user accepts the perms, should we be able to modify the AUDIO settings on our own ?